pages/user guides/discover_nmap.md

```diff
@@ -8,58 +8,53 @@ permalink: discover-nmap.html
+With Nmap's powerful capabilities and good [documentation](https://nmap.org/book/host-discovery-find-ips.html), you can identify all the hosts you have running on your network, open ports, and running services. It is possible to discover those places which are often hard to reach.
+
+Nmap is capable of producing its output in an XML file. It allows you to inspect the raw scan output before sending it to the Tidal Migrations API with [Tidal Tools](https://get.tidal.sh). Once uploaded to the Tidal Migrations Platform, you will be able to visualize your network devices, track your complete server inventory, and build on this data with other discovery methods. This is how you can make informed decisions on your cloud migration path.
 
 
 ## Using Nmap with Tidal Tools
- 
-By leveraging the power of **Tidal Tools** you can send the output generated by Nmap to your Tidal Migrations account.
- 
+By leveraging the power of **Tidal Tools**, you can send the output generated by Nmap to your Tidal Migrations account.
 1. Install [Tidal Tools](https://guides.tidalmg.com/tidal-tools.html)
-2. [Login to Tidal Tools](https://guides.tidalmg.com/tidal-tools.html#login) with **tidal login**
-3. Run Nmap ([Need to install Nmap?](#installing-nmap)) with the flags of your choosing and save the output to an XML file. For example, 
+2. Connect Tidal Tools and your Tidal Migrations account with [tidal login](https://guides.tidalmg.com/tidal-tools.html#login).
+3. Run Nmap with the flags of your choosing and save the output to an XML file. For example,
     ```
-    sudo nmap -sS -p80,443,8080,8443,1433,1521,27017 <ip-address> -oX my-network.xml
+    sudo nmap -sV -p80,443,8080,8443,1433,1521,27017 <ip-address/range> -oX my-network.xml
     ```
-    _Note: the **-sS** flag will enable TCP SYN port scan and the **-oX** specifies the output as a XML file. [Want more scanning options?](#nmap-usage)_
-    {% include tip.html content="Be sure to replace <ip-address> with the CIDR range for the network you would like to scan, ex. `10.0.0.0/24`" %}
+    [Go here to install Nmap](#installing-nmap).
+    
+    _Note: the **-sV** flag will attempt to determine the version of the service running on port and the **-oX** specifies the output as an XML file. [Want more scanning options?](#nmap-usage)_
 
+    {% include tip.html content="Be sure to replace <ip-address/range> with the CIDR range for the network you would like to scan, ex. `10.0.0.0/24`" %}
+ 
 4. Run this Tidal Tools command to upload your previously generated Nmap output to your Tidal Migrations account 
 ```
 tidal sync nmap <filename.XML> 
```

Comment:
Let's change it to `tidal sync nmap my-network.xml` since previously we showed how to create a file `my-network.xml` with Nmap. And also those angle brackets wrapping `filename.XML` look so Java generics to me :rofl:

Reply:
yes, good point. 👍